ReferenceRetailer/reference/src/main/AndroidManifest.xml

```diff
@@ -64,7 +65,7 @@
             android:screenOrientation="sensorPortrait"
             android:exported="false"
             android:label="@string/app_name"
-            android:windowSoftInputMode="stateVisible|adjustUnspecified">
+            android:windowSoftInputMode="stateVisible|adjustPan">
```

Comment:
In this way, the window won't resize itself, and the button won't display above keyboard again.